src/app/component/eco-news/components/create-news/create-news.component.ts

```diff
@@ -36,6 +36,7 @@ export class CreateNewsComponent implements OnInit {
   public isLink = false;
   public formData: FormGroup;
   public isArrayEmpty = true;
+  private author: string = localStorage.getItem('name');
```

Comment:
it should be public, as you using this in template file

Reply:
fixed